orm/src/test/java/org/hibernate/search/test/jpa/SpatialQueryingJPATest.java

```diff
@@ -74,12 +74,12 @@ public void testDistanceProjection() throws Exception {
 		Object[] fourthResult = (Object[]) results.get( 3 );
 		Object[] fifthResult = (Object[]) results.get( 4 );
 		Object[] sixthResult = (Object[]) results.get( 5 );
-		Assert.assertEquals( 0.0, (Double) firstResult[1], 0.0001 );
-		Assert.assertEquals( 10.1582, (Double) secondResult[1], 0.0001 );
-		Assert.assertEquals( 11.1195, (Double) thirdResult[1], 0.0001 );
-		Assert.assertEquals( 15.0636, (Double) fourthResult[1], 0.0001 );
-		Assert.assertEquals( 22.239, (Double) fifthResult[1], 0.001 );
-		Assert.assertEquals( 24.446, (Double) sixthResult[1], 0.001 );
+		Assert.assertEquals( 0.0, (Double) firstResult[1], 0.01 );
```

Comment:
Nope. They probably use different algorithms to calculate the distance. Information about it are sparse. It was far worse when I used the distanceInKm function and became acceptable with the arcDistanceInKm function.